src/Views/AlertButton.js

```diff
@@ -0,0 +1,25 @@
+import View from './View.js'
+
+const AlertButton = new View()
+
+AlertButton.setUp = function (el) {
+  this.el = el
+  this.render().bindEvents()
+  return this
+}
+
+AlertButton.render = function () {
+  this.el.innerText = 'Click Me!'
+  return this
+}
+
+AlertButton.bindEvents = function () {
+  this.el.addEventListener('click', this.nicotinamideAdenineDinucleotide)
+  return this
+}
+
+AlertButton.nicotinamideAdenineDinucleotide = function () {
```

Comment:
```suggestion
AlertButton.nicotinamideAdenineDinucleotide = function () {
```

`nicotinamideAdenineDinucleotide`이벤트 명이 적절하지 않아보여요 😅. 보다 직관적인 이름으로 변경되어야 할것같아요.

Reply:
b99322bf48381d759d3014c507e9d2cd294b3fb6
더 멋진 이름으로 변경했습니다!